__tests__/pathfinding.service.test.js

```diff
@@ -132,11 +112,80 @@ describe("Find the shortest path between two locations/POI's", () => {
     const path = findPathBetweenPOIs(
       POIInfo.find(
         ({ buildingId, displayName, level }) =>
-          displayName === "Women's Bathroom" && level === 8
+          displayName === "Women's Bathroom" &&
+          level === 8 &&
+          buildingId === "H"
       ),
-      POIInfo.find(({ displayName }) => displayName === "H961-17")
+      POIInfo.find(({ displayName }) => displayName === "H961-17"),
+      false
     );
     expect(path.length).toBe(2);
     expect(path).toEqual(testData.h8WomensBathroomToH9117);
   });
+
+  it("should return null when no path is found", () => {
+    // store the initial nodes
+    const initialNodes = _.cloneDeep(
+      constants.buildingFloors.find(
+        (floor) => floor.buildingId === "H" && floor.level === 9
+      ).travelNodes
+    );
+    constants.buildingFloors.find(
+      (floor) => floor.buildingId === "H" && floor.level === 9
+    ).travelNodes = testData.brokenGraph;
+    const shortest = findPathOnFloor(
+      POIInfo.find(({ displayName }) => displayName === "H961-19"),
+      POIInfo.find(({ displayName }) => displayName === "H911"),
+      false
+    );
+
+    expect(shortest).toBeNull();
+    // reset the initial nodes
+    constants.buildingFloors.find(
+      (floor) => floor.buildingId === "H" && floor.level === 9
+    ).travelNodes = initialNodes;
+  });
+
+  it("when accessibility mode is enabled, should return the most accessible path between two given POI's in Hall and JMSB. Should take the ramp in MB.", () => {
```

Comment:
Nice description! But please modify the test name to match the `it` declaration. For example, `it("should...")`, etc.